Source/WarpX.cpp

```diff
@@ -216,6 +218,13 @@ WarpX::WarpX ()
     pml.resize(nlevs_max);
     costs.resize(nlevs_max);
 
+
+    if (em_solver_medium == 1) {
```

Comment:
As elsewhere, can you use the enum value instead of 1? Same for below, line 629

Reply:
done